Automattic/jetpack
# Fix Blogroll block icon

## Proposed changes:

https://github.com/Automattic/jetpack/pull/32815 moved the markup of the _Blogroll_ block icon from `icon.js` to `block.json`. While that PR was in progress, the icon was updated. This PR makes sure we have the latest version in `block.json`

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Enable beta blocks in the Jetpack constants section if you're using a JN site, or by adding `define( 'JETPACK_BLOCKS_VARIATION', 'beta' );` to your `wp-config.php` otherwise.
- Check that the icon for the block is this one:
<img width="119" alt="Screenshot 2023-09-14 at 9 47 22 AM" src="https://github.com/Automattic/jetpack/assets/1620183/29eec8f7-a302-4bb1-bfd1-03417779e726">
